basicMysql/register/deleteotp.php

```diff
@@ -0,0 +1,45 @@
+<?php
+	session_start();
+	if(!isset($_SESSION['login_user'])){
+		header("location: ../login/login.php");
+	}
```

Comment:
Two spaces should be provided according to the convention extra spaces needs to be removed.